src/EventSubscriber/IconOptionsSubscriber.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_paragraphs\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Drupal\oe_paragraphs\Event\IconOptionsEvent;
+
+/**
+ * Provides options for the icon field.
+ */
+class IconOptionsSubscriber implements EventSubscriberInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents(): array {
+    return [
+      IconOptionsEvent::class => 'getIconOptions',
+    ];
+  }
+
+  /**
+   * Gets the icon options.
+   *
+   * @param \Drupal\custom_events\Event\IconOptionsEvent $event
+   *   Allowed format event object.
+   */
+  public function getIconOptions(IconOptionsEvent $event): void {
+    $event->setIconOptions([
```

Comment:
Should icons be as presented in oe_theme/dist/ec/images/icons? If yes, then this list isn't the same as icons provided there.

Reply:
We are not changing the list of icons in this ticket.